### PR TITLE
wappalyzer: Adapt to multiple tech files and separate categories file

### PR DESCRIPTION
**Pull request by kingthorin**
- Update add-on code to handle multiple technology files, and a separate categories file per upstream changes.
- Update UnitTests to accommodate the new functionality/setup.
- Added change note to CHANGELOG.
- Updated scheduled GitHub action workflow to update things based on the new setup.

Fixes zaproxy/zaproxy#6784

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

File: addOns/wappalyzer/src/main/java/org/zaproxy/zap/extension/wappalyzer/WappalyzerJsonParser.java [hunk 1]
```diff
@@ -74,100 +74,99 @@ public WappalyzerJsonParser() {
...
-        WappalyzerData result = new WappalyzerData();
-
+    private void parseCategories(WappalyzerData wappalyzerData, String jsonStr) {
         try {
             JSONObject json = JSONObject.fromObject(jsonStr);
 
-            JSONObject cats = json.getJSONObject("categories");
-
-            for (Object cat : cats.entrySet()) {
+            logger.debug("There seems to be: {} categories to load", json.entrySet().size());
```
Comment (ricekot) [inline addOns/wappalyzer/src/main/java/org/zaproxy/zap/extension/wappalyzer/WappalyzerJsonParser.java:106]: nit: `seem`

**Comment by kingthorin** [comment]
Tweaked

**Review by ricekot — APPROVED** [review]
Thanks!

**Pull request merged**